evaluate/eval.py

```diff
@@ -70,11 +70,14 @@ def eval_report(self):
             prec, rec, rep_recall, f1, rep_f1 = self.evaluate_repairs()
             report = "Precision = %.2f, Recall = %.2f, Repairing Recall = %.2f, F1 = %.2f, Repairing F1 = %.2f, Detected Errors = %d, Total Errors = %d, Correct Repairs = %d, Total Repairs = %d, Total Repairs (Grdth present) = %d" % (
                       prec, rec, rep_recall, f1, rep_f1, self.detected_errors, self.total_errors, self.correct_repairs, self.total_repairs, self.total_repairs_grdt)
+            report_list = [prec, rec, rep_recall, f1, rep_f1, self.detected_errors, self.total_errors,
+                           self.correct_repairs, self.total_repairs, self.total_repairs_grdt]
         except Exception as e:
             report = "ERROR generating evaluation report: %s"%str(e)
+            report_list = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
I think -1 is better NA values for accuracy

Reply:
changed